### PR TITLE
Dropout translation

### PR DESCRIPTION
**Pull request by varunjain99**
1) Translates dropout for test mode
2) Changes condition for batch normalization throw so that throws only when it can't produce necessary outputs
@bddppq @yinghai @Maratyszcza

### CODE REVIEW

File: onnx_xla/translations/translate_batch_normalization.cc [hunk 1]
```diff
@@ -7,9 +7,14 @@ namespace onnx_xla {
 onnxStatus translateBatchNormalization(const Node& n,
                                        XlaBuilder& builder,
                                        ValueOpMap& valueToOp) {
-  if (n.outputs().size() > 1 || n.hasAttribute(kmomentum) ||
-      n.hasAttribute(Symbol("spatial"))) {  // TODO: ENFORCE
-    throw std::runtime_error("Only test mode of BatchNormalization supported");
+  // If requires output of train mode, throw
+  if (n.outputs().size() > 1) {
```
Comment (bddppq) [inline onnx_xla/translations/translate_batch_normalization.cc]: 1. doesn't seem to need this `if`
2. keeping the attribute check is better IMO

Comment (varunjain99) [reply onnx_xla/translations/translate_batch_normalization.cc]: Removing unneeded if and bringing back attribute checks
